ruby/ql/lib/codeql/ruby/ast/Constant.qll

```diff
@@ -208,6 +208,11 @@ class ConstantAccess extends Expr, TConstantAccess {
    */
   predicate hasGlobalScope() { none() }
 
+  /**
+   * Gets a fully qualified name for this constant.
+   */
+  string getAQualifiedName() { none() }
```

Comment:
The change looks fine; however `getAQualifiedName` is not used in this pull request so I wonder why you are changing this here.

Reply:
Ah, I was using this in an earlier revision of https://github.com/github/codeql-model-manager/pull/5 to get qualified names for general `ConstantAccess`es, but changed how this is implemented so this predicate isn't required. I'll drop it from this PR.